v2/client.go

```diff
@@ -139,18 +146,29 @@ func requestContentType(ct string) requestOption {
 	}
 }
 
-// buildURL builds a url to /api/v2/... using the given pathElements. It
-// url escapes each path element, so the caller doesn't need to worry about
-// that.
+// buildURL builds a url to /api/v2/... using the given pathElements.
+// It url escapes each path element, so the caller doesn't need to worry about that.
 func (c *Client) buildURL(pathElements ...any) *url.URL {
```

Comment:
Note for ourselves that can be addressed outside of this PR: might be good to have some basic regression unit tests around this method (namely for our expectations around path escaping)